cli/src/pcluster/utils.py

```diff
@@ -1149,6 +1157,14 @@ def vcpus_count(self) -> int:
 
         return vcpus
 
+    def vcpus(self, disable_simultaneous_multithreading):
```

Comment:
vcpus of instance_type info should return the right number of vcpus, not the one adapted according to simultaneous_multithreading. Let's restore previous behaviour.

Reply:
Done